### ISSUE TITLE
Refactor interface

### ISSUE THREAD

**Issue by misterrevo** [karunaproject/be#53]
Think if IErrorResponse is needed

**Comment by misterrevo** [comment]
@BarSwi we have exceptions which use this interface, its to discuss if we realy need to remove it

**Comment by BarSwi** [comment]
> @BarSwi we have exceptions which use this interface, its to discuss if we realy need to remove it

Yes, but this interface is nowhere provided as method argument, therefore polymorphism does not apply. Those exceptions only implement the interface but interface itself is not used anywhere, so it looks like a leftover right now in my opinion.
